solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -399,8 +512,52 @@
                 "capacity": 1
             },
             "properties": {
-                "location": "[variables('location')]"
-            }
+                "location": "[variables('location')]",
+                "routing": {
+                    "endpoints": {
+                        "eventHubs": [
+                            {
+                                "connectionString": "[concat('Endpoint=sb://', parameters('eventHubNamespaceName'), '.servicebus.windows.net/;SharedAccessKeyName=', parameters('eventHubAuthorizationName'), ';SharedAccessKey=', listkeys(variables('eventHubRouteKeyResource'), variables('eventHubApiVersion')).primaryKey, ';EntityPath=',  parameters('eventHubName'))]",
```

Comment:
Will `.servicebus.windows.net` be different for china cloud?

Reply:
How can I find this out? The reason I wrote it like this is the endpoint returned by listkeys is not the exact one for the connection string. Per the event hub programming guide the endpoint is:
Endpoint=sb://[namespace].servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=[key]
No mention of change for different regions, not sure if that would be stated or not.

Reply:
Table is from [this](https://msdn.microsoft.com/en-us/library/azure/dn578439.aspx?f=255&MSPPError=-2147217396#About%20this%20Document) link

Service Type | Global Service URI | China Service URI
-- | -- | --
Windows Azure - General | *.windows.net | *.ch *.chinacloudapi.cn
Windows Azure Compute | *.cloudapp.net | *.chinacloudapp.cn
Windows Azure Storage | *.blob.core.windows.net*.queue.core.windows.net*.table.core.windows.net | *.blob.core.chinacloudapi.cn*.queue.core.chinacloudapi.cn*.table.core.chinacloudapi.cn
Windows Azure Service Management | https://management.core.windows.net | https://management.core.chinacloudapi.cn/
SQL Database | *.database.windows.net | *.database.chinacloudapi.cn
Windows Azure Management Portal | http://manage.windowsazure.com | http://manage.windowsazure.cn
SQL Azure Database Management API | https://management.database.windows.netSee About the SQL Azure Database Management API | https://management.database.chinacloudapi.cn
Service Bus | *.servicebus.windows.net | *.servicebus.chinacloudapi.cn
SQL Database Import/Export service mapping endpoints |   | China East: https://sh1prod-dacsvc.chinacloudapp.cn/dacwebservice.svcChina North: https://bj1prod-dacsvc.chinacloudapp.cn/dacwebservice.svc